docs/cloud-dev-env.md

```diff
@@ -0,0 +1,31 @@
+# Cloud Development Environments
+
+Altis has experimental built-in support for [GitHub Codespaces](https://github.com/features/codespaces), a Cloud-based development environment, that makes it possible to spin up a complete development environment in seconds within your browser, with a full editor based on Visual Studio Code and a fully-featured environment for your project, without having to install anything on your local machine.
+
+This is useful for many reasons:
+    - Bootstrap a full development environment in seconds.
+    - Zero local installation required.
+    - Standardized and highly configurable environment setup.
+    - Powerful and scalable development environment.
+    - Possibility of sharing preview environments with colleagues.
+
+## How to start
+
+1. **Enable Codespaces support for your GitHub organization**
+In order for Codespaces to become available to contributors of a specific repository, the GitHub organization administrator needs to enable support for Codespaces for such contributors. Refer to the official [Enabling Codespaces for your organization](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/enabling-codespaces-for-your-organization) and [Managing billing for Codespaces in your organization](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-billing-for-codespaces-in-your-organization) guides for more information.
+
+2. **Bootstrap the configuration**
+After Codespaces is activated, you'll need to bootstrap the development container configuration using `composer dev-tools bootstrap codespaces`. See the [documentation on devcontainers](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/introduction-to-dev-containers) for way to customize your container further.
+
+3. **Start a new container**
+Once the devcontainer configuration is committed, you'll be able to start a new Codespaces environment at https://github.com/codespaces/new , or from the "Codespaces" tab when you click on "Code" button where you typically get your repository checkout link.
+
+4. **Start development!**
+Once the codespace is ready, you'll be able to use the editor-in-browser to manage the project code, and the terminal to start the integrated local server environment as you would if you installed it locally, using `composer server start`, and to pull and push code in a fully-featured Debian-based environment.
```

Comment:
When you get a long sentence like this it’s ok to put some full stops in rather than always using commas, rather than cramming all the info into a single sentence it can improve readability